cpmonitor/models.py

```diff
@@ -199,48 +199,54 @@ class Meta:
-        DELAYED = 3, "zurückgestellt / verzögert (orange)"
-        INSUFFICIENT = 4, "nicht ausreichend / in Teilen gescheitert (rot)"
-        FAILED = 5, "gescheitert (rot)"
-        MISSING = 6, "fehlt im Plan (rot)"
-
-    execution_assessment = models.IntegerField(
-        "Bewertung des Umsetzungsstandes",
-        choices=ExecutionAssessment.choices,
-        default=ExecutionAssessment.UNKNOWN,
+    class ExecutionStatus(models.IntegerChoices):
+        UNKNOWN = 0, "unbekannt / ungeplant"
+        AS_PLANNED = 2, "geplant / in Arbeit"
+        COMPLETE = 4, "abgeschlossen"
+        DELAYED = 6, "verzögert / fehlt"
+        FAILED = 8, "gescheitert"
+
+    execution_status = models.IntegerField(
+        "Umsetzungsstand",
+        choices=ExecutionStatus.choices,
+        default=ExecutionStatus.UNKNOWN,
         help_text="""
             <p>Bei Maßnahmen: Wird/wurde die Maßnahme wie geplant umgesetzt?</p>
+            <dl>
+                <dt>unbekannt / ungeplant</dt><dd><ul>
+                    <li>Keine Infos vorhanden</li>
+                    <li>Es gibt einen unkonkreten Beschluss</li>
+                </ul></dd>
+                <dt>geplant / in Arbeit</dt><dd><ul>
+                    <li>Finden wir erstmal gut: Dinge werden bearbeitet.</li>
+                    <li>Maßnahmen, die im Zeitplan sind (sowohl begonnen als auch in Planung).</li>
+                    <li>Maßnahmen die in Umsetzung sind.</li>
+                    <li>Maßnahmen für die es einen positiven Beschluss + Zeitplan (Konkretisierung) gibt.</li>
+                </ul></dd>
+                <dt>abgeschlossen</dt><dd><ul>
+                    <li>Top und im Plan fertig.</li>
+                </ul></dd>
+                <dt>verzögert / fehlt</dt><dd><ul>
+                    <li>Maßnahmen, die nicht im Zeitplan sind.</li>
+                    <li>Maßnahmen, die im Prinzip noch vervollständigt werden können.</li>
+                    <li>Maßnahmen, die im KAP nicht aufgeführt sind, (und nicht bearbeitet weden)</li>
+                    <li>Im Prinzip könnten sie aber noch angegangen werden.</li>
+                </ul></dd>
+                <dt>gescheitert</dt><dd><ul>
+                    <li>Kann niemals mehr geschafft werden (Eiche ist gefällt, Moor ist mit einer Autobahn überbaut)</li>
+                    <li>Sowohl für Maßnahmen aus dem KAP, als auch Dinge, die gar nich im KAP aufgeführt waren.</li>
+                </ul></dd>
+            </dl>
             <p>Bei Sektoren / Maßnahmengruppen:</p>
             <p>Wenn hier "unbekannt" ausgewählt wird, werden die Umsetzungsstände der Maßnahmen in diesem Sektor / dieser Gruppe zusammengefasst.</p>
             <p>Bei anderen Auswahlen wird diese Zusammenfassung überschrieben. Das sollte nur passieren, wenn sie unpassend oder irreführend ist.</p>
         """,
     )
 
     execution_justification = models.TextField(
-        "Begründung der Bewertung",
+        "Begründung Umsetzungsstand",
```

Comment:
I'd leave "Begründung der Bewertung" as before, because that's what it is really. We are not explaining the implemetation status (meaning sth like "what went wrong, what are the reasons"), but we are explaining our assessment (meaning sth like "it's delayed, because the mayor said it would be completed in january and now it's august")

Reply:
Isnt "Bewertung" what we want to avoid? Besides, "Umsetzungsstand" is simply the field that this field refers to. The help-text explains, what is to be entered in the admin.

Reply:
I don't think we want to avoid a "Bewertung" altogether. The question was just what to assess: quality or progress. But I see your point and I am undecided.